test/cri-containerd/test-images/jobcontainer_createvhd/main.go

```diff
@@ -1,3 +1,6 @@
+//go:build windows
+// +build windows
```

Comment:
Do we need to support the old build tags if we're on 1.17 in /test also?

Reply:
(Same comment for the other test-image binaries)

Reply:
I think running `go fix` in 1.18 will remove/upgrade all of those
Ill save removing them for when we upgrade to 1.18, since this pr is so massive

Reply:
sgtm